src/routes/app.js

```diff
@@ -40,6 +40,89 @@ app.extend({
 
     if (__DEV__) {
       window.Perf = require('react-addons-perf')
+      window.Paused = false
+      window.ClearRecordings = () => {
```

Comment:
I think there should be an object here.  Like, window.guesstimateRecorder or something.  These commands should be on that recorder.  Tying them all directly to widow seems messy to me.